### PR TITLE
Added the option to change the wakeup interval of a device from an item

### PR DESCRIPTION
**Pull request by bodiroga**
@cdjackson , as discussed by email, I have added the option to set the wakeup interval of a device using an item. Few changes have been made:
- New ZWaveWakeUpConverter class has been added.
- ZWaveConverterHandler has been updated to support the new converter.

I am very new to the git and github stuff, I hope I did not make any mistake.

Aitor

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/converter/ZWaveWakeUpConverter.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    public void receiveCommand(Item item, Command command, ZWaveNode node, ZWaveWakeUpCommandClass commandClass,
+            int endpointId, Map<String, String> arguments) {
+        ZWaveCommandConverter<?, ?> converter = this.getCommandConverter(command.getClass());
+        if (converter == null) {
+            logger.warn("NODE {}: No converter found for item={}, type={}, endpoint={}, ignoring command.",
+                    node.getNodeId(), item.getName(), command.getClass().getSimpleName(), endpointId);
+            return;
+        }
+
+        ZWaveProductDatabase database = new ZWaveProductDatabase();
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/converter/ZWaveWakeUpConverter.java]: This section can be deleted as the database is not required

**Comment by cdjackson** [comment]
Looks good - just one small update to make...

I still don't know that this is necessary - I need to dig out the battery life estimator I wrote a while back... Anyway, for OH1 there's no downside and we'll no doubt have the discussion again for OH2 :)

**Comment by bodiroga** [comment]
Required changes done Chris, I suppose that now I need to squash both commits into a single one, right?

**Comment by cdjackson** [comment]
> I suppose that now I need to squash both commits into a single one, right?

Yes please :)

**Comment by bodiroga** [comment]
Done ;)

**Comment by cdjackson** [comment]
Thanks.

**Comment by cdjackson** [comment]
Thanks

**Pull request merged**